channels/pjsip/dialplan_functions_doc.xml

```diff
@@ -55,6 +55,43 @@
 		</description>
 	</application>
 
+	<application name="PJSIPNotify" language="en_US">
+		<synopsis>
+			Send a NOTIFY to either an arbitrary URI, or inside a SIP dialog.
+		</synopsis>
+		<syntax>
+			<parameter name="to" required="false">
+				<para>Abritrary URI to which to send the NOTIFY.  If none is specified, send inside
+				the SIP dialog for the current channel.</para>
+			</parameter>
+			<parameter name="headers" required="true">
+				<para>Either a pre-configured option or a list of headers to send in the NOTIFY.</para>
+			</parameter>
+		</syntax>
+		<description>
+			<para>
+			Sends a NOTIFY to a specified URI, or if none provided, within the current SIP dialog for the
+			current channel.  The headers can either be set to either an entry configured in pjsip_notify.conf
+			or specified as a list of key value pairs.
+			</para>
+			<warning><para>
+			To send a NOTIFY to a specified URI, a default_outbound_endpoint must be configured.  This
+			endpoint determines the message contact.
+			</para></warning>
+			<para>
+			</para>
+			<example title="Send a NOTIFY with Event and Data headers in current dialog">
+			same = n,PJSIPNotify(,&amp;Event=Test&amp;Data=Fun)
```

Comment:
"Data" here is really confusing.  It kinda implies you're setting the body of the Notify instead of a SIP header.  In pjsip_notify.conf, you use X-Data which is better.  You should use it here as well.